Fix Conflicting Static File Serving in Backend Server

This issue identifies an error when running the backend server with the `APP_DIST_DIR` environment variable specified, as shown below:

```
{"level":50,"time":1752457394182,"pid":43109,"hostname":"salmonsashimi.local","err":{"type":"FastifyError","message":"The decorator 'sendFile' has already been added!","stack":"FastifyError: The decorator 'sendFile' has already been added!\n    at decorateConstructor (/home/user/festabook/node_modules/.pnpm/fastify@5.4.0/node_modules/fastify/lib/decorate.js:51:11)\n    at Object.decorateReply (/home/user/festabook/node_modules/.pnpm/fastify@5.4.0/node_modules/fastify/lib/decorate.js:125:3)\n    at fastifyStatic (/home/user/festabook/node_modules/.pnpm/@fastify+static@8.2.0/node_modules/@fastify/static/index.js:79:13)\n    at Plugin.exec (/home/user/festabook/node_modules/.pnpm/avvio@9.1.0/node_modules/avvio/lib/plugin.js:125:28)\n    at Boot._loadPlugin (/home/user/festabook/node_modules/.pnpm/avvio@9.1.0/node_modules/avvio/boot.js:432:10)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","code":"FST_ERR_DEC_ALREADY_PRESENT","name":"FastifyError","statusCode":500},"msg":"The decorator 'sendFile' has already been added!"}
/home/user/festabook/server:
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command failed with exit code 1: jiti src/index.ts
```

This issue likely occurs because `@fastify/static` cannot be registered more than once unless the `decorateReply` option is set to `false` on subsequent registrations, as explained in [their guide](https://github.com/fastify/fastify-static#multiple-prefixed-roots).

--- a/server/src/index.ts
+++ b/server/src/index.ts
@@ -17,6 +17,7 @@ if (process.env.APP_DIST_DIR) {
   fastify.register(fastifyStatic, {
     root: process.env.APP_DIST_DIR,
     prefix: "/",
+    decorateReply: false,
   });
 }
 

